plugins/modules/dedicated_server_intervention.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.module_utils.basic import AnsibleModule
+
+DOCUMENTATION = '''
+---
+module: dedicated_server_intervention
+short_description: Enable or disable ovh intervention on a dedicated server
+description:
+    - Enable or disable ovh intervention on a dedicated server
+author: Synthesio SRE Team
+requirements:
+    - ovh >= 0.5.0
+options:
+    service_name:
+        required: true
+        description: The service_name
```

Comment:
No need for snake case: 
```suggestion
        description: The service name
```